SDWebImage/SDImageCache.m

```diff
@@ -106,14 +106,6 @@ - (void)dealloc {
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
-- (void)checkIfQueueIsIOQueue {
-    const char *currentQueueLabel = dispatch_queue_get_label(DISPATCH_CURRENT_QUEUE_LABEL);
-    const char *ioQueueLabel = dispatch_queue_get_label(self.ioQueue);
-    if (strcmp(currentQueueLabel, ioQueueLabel) != 0) {
-        NSLog(@"This method should be called from the ioQueue");
```

Comment:
Actually, whenever you call `-[SDImageCache storeImageDataToDisk:forKey:]` which a our public API. This log is printed...Really awkward :)